src/components/button-link/button-link.module.css

```diff
@@ -52,6 +52,10 @@
     color: var(--token-color-foreground-disabled);
     cursor: not-allowed;
   }
+
+  &:hover:not(:disabled) {
+    text-decoration: underline;
+  }
```

Comment:
🚧 I believe we want this underline to show all the time based on the HDS component handles rendering links that look like buttons:

![image](https://user-images.githubusercontent.com/43934258/170617687-d2487b44-bfb7-4a77-96a3-5812e1ef7be6.png)

I was involved in the RFC that held the discussion about this, and the rationale is so that there is always a visual distinction between buttons and links-that-look-like-buttons.

Reply:
Neat, apologies for missing this earlier, thanks for re-linking the screenshot! 👍

Reply:
✅ Here's the updated appearance:

<img width="901" alt="CleanShot 2022-05-27 at 08 43 30@2x" src="https://user-images.githubusercontent.com/4624598/170701854-a69c8380-31ac-4ee0-a7ec-7da3f63e48c8.png">